packages/bolt-matrix/mod.ts

```diff
@@ -11,7 +11,7 @@ import {
 import { coreToMessage, onEvent } from './events.ts';
 
 type MatrixConfig = {
-	accessToken: string;
+	appserviceUrl: string;
```

Comment:
any reason for this change?

Reply:
`accessToken` is not used

`appserviceUrl` is where the homeserver should ping the appservice at, not the homeserver's url.

Reply:
makes sense, didn’t see that when i first looked at this